linux/action.sh

```diff
@@ -34,7 +34,7 @@ function qualify_distro() {
         printf "ubuntu:jammy"
         ;;
     "suse")
-        printf "registry.suse.com/suse/sle15:latest"
+        printf "registry.suse.com/suse/sle15:15.3"
```

Comment:
Can we make this renovate-aware? Should not be too hard with a couple regexes.

We probably won't merge updates right away, but if there is a PR open it is easy for us to not forget about it.

Reply:
I think so , I will create an issue since at least to me it would take some time to do it

Reply:
https://github.com/newrelic/integrations-pkg-test-action/issues/32